src/hotspot/share/services/diagnosticCommand.hpp

```diff
@@ -858,6 +858,28 @@ class SystemDictionaryDCmd : public DCmdWithParser {
   virtual void execute(DCmdSource source, TRAPS);
 };
 
+class ClassesDCmd : public DCmdWithParser {
+protected:
+  DCmdArgument<bool> _verbose;
+public:
+  ClassesDCmd(outputStream* output, bool heap);
+  static const char* name() {
+    return "VM.classes";
+  }
+  static const char* description() {
+    return "Prints list of all loaded java classes";
+  }
+  static const char* impact() {
+      return "Medium: Depends on Java content.";
```

Comment:
Thanks for reviews! Since ClassHierarchyDCmd uses `"Medium: Depends on number of loaded classes."`, so I'm going to change the impact description but keeping as `Medium` level.